contracts/DADBridgeWrapper.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.9;
+
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DADBridgeWrapper is AccessControl {
+    using SafeERC20 for IERC20;
+
+    bytes32 public constant ROLE_ADMIN = keccak256("ROLE_ADMIN");
+
+    IERC20 public immutable bntToken;
+    IERC20 public immutable ethBntToken;
+
+    uint256 public totalSupply;
```

Comment:
these are old conventions. not the end of the world but if it's not too time consuming, I'd still go for the latest we use (private state vars, custom getters, underscores for privates etc.) - not critical

Reply:
Fixed.